servers/lib/test/e2e/app.e2e.spec.ts

```diff
@@ -1,14 +1,10 @@
+import { describe, it, expect } from '@jest/globals';
 import { Test, TestingModule } from '@nestjs/testing';
 import { INestApplication } from '@nestjs/common';
-import * as request from 'supertest';
+import request from 'supertest';
 import fetch from 'cross-fetch';
-import {
-  ApolloClient,
-  DocumentNode,
-  HttpLink,
-  InMemoryCache,
-  gql,
-} from '@apollo/client';
+import { ApolloClient, DocumentNode, InMemoryCache, gql} from "@apollo/client/core/core.cjs";
```

Comment:
can we import esm modules of the library with `mjs` extension?

Reply:
I had to change it, since @apollo client isn't esm yet